api/views.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import generics, permissions
+from .serializers import BookSerializer
+from book.models import Book
+
+
+class BookList(generics.ListAPIView):
+    serializer_class = BookSerializer
+
+    def get_queryset(self):
+        user = self.request.user
+        return Book.objects.filter(user=user).order_by("title")
+
+
+class BookListCreate(generics.ListCreateAPIView):
+    serializer_class = BookSerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+    def get_queryset(self):
+        user = self.request.user
+        return Book.objects.filter(user=user).order_by("title")
```

Comment:
Why do you need both of these views?

Reply:
I think it is correct if read all books list can any, but create only authenticated

Reply:
Then I'd suggest using `CreateAPIView` instead of `ListCreateAPIView`.